engine/settings/base.py

```diff
@@ -569,6 +561,18 @@ class BrokerTypes:
 
 EXTRA_MESSAGING_BACKENDS = []
 
+# Email messaging backend
+EMAIL_BACKEND = "django.core.mail.backends.smtp.EmailBackend"
+EMAIL_HOST = os.getenv("EMAIL_HOST")
+EMAIL_HOST_USER = os.getenv("EMAIL_HOST_USER")
+EMAIL_HOST_PASSWORD = os.getenv("EMAIL_HOST_PASSWORD")
+EMAIL_PORT = getenv_integer("EMAIL_PORT", 587)
+EMAIL_USE_TLS = getenv_boolean("EMAIL_USE_TLS", True)
+DEFAULT_FROM_EMAIL = os.getenv("DEFAULT_FROM_EMAIL")
+
+if FEATURE_EMAIL_INTEGRATION_ENABLED:
+    EXTRA_MESSAGING_BACKENDS = [("apps.email.backend.EmailBackend", 587)]
```

Comment:
I recommend to increment the id to match the current code style
however the idea with 587 is smart:)

Reply:
Makes sense, I changed it to use 8 as backend id https://github.com/grafana/oncall/pull/621/commits/425d821cd83f08611092a90b06f2ba9b924d5ab2